beacon-chain/sync/backfill/service.go

```diff
@@ -336,10 +339,6 @@ func (s *Service) initBatches() error {
 	return nil
 }
 
-func (s *Service) downscore(b batch) {
```

Comment:
I kinda like this function name, it makes sense. it makes more sense than just .BadResponsesScorer().Increment(ib.blockPid)

Reply:
or maybe a different name for this

Reply:
Fixed in 9d2bc52231ce7fd0a0e22b09add06fc95b38aefd.